MFVNeutralino/python/NtupleCommon.py

```diff
@@ -1,18 +1,22 @@
 from JMTucker.Tools.CMSSWTools import *
 from JMTucker.Tools.Year import year
 
-#ntuple_version_ = 'ULV9_dileptrig'
-#ntuple_version_ = 'ULV9_trigstudy_'
-#ntuple_version_ = 'UL16APVV9_trigstudy_'
 ntuple_version_ = 'OnnormdzULV30'
-lsp_id = -1 # should do that in a smarter way would be -1 if not MET
-use_btag_triggers = True
+lsp_id = -1 #1000009 # should do that in a smarter way; currently for stop if not -1
+use_btag_triggers = False
 use_MET_triggers = False
+use_Muon_triggers = True
+use_Electron_triggers = False
+use_DisplacedLepton_triggers = False
 if use_btag_triggers : 
     ntuple_version_ += "B" # for "Btag triggers"; also includes DisplacedDijet triggers
 elif use_MET_triggers :
     lsp_id = 1000021 # should do that in a smarter way would be -1 if not MET
     ntuple_version_ += "MET"
+elif use_Muon_triggers :
+    ntuple_version_ += "LepMu"
+elif use_Electron_triggers :
+    ntuple_version_ += "LepEle"
 ntuple_version_use = ntuple_version_ + 'm'
```

Comment:
Since everyone has to set these things, should we find some smarter way to set ntuple_version and which triggers to use, that we don't all have to change and then be careful not to commit? We could read it from an environment variable that we set in our .bashrc or some other file, or import from a file that we don't track...? Or have it as a command line argument, but maybe that would be a pain to propagate everywhere. What do people think? @pkotamnives @abbywarden @duquale